site/public/ts/utils/tools/SelectionTool.ts

```diff
@@ -122,24 +138,20 @@ export class SelectionTool extends Tool {
                 // Go through each object and see if it's within
                 //  the selection box
                 var objects = this.designer.getObjects();
+                let new_objs = false;
                 for (let obj of objects) {
                     // Check if object is in box
                     if (TransformContains(box, obj.getTransform())) {
                         // Add to selections if not already selected
-                        if (!this.selections.includes(obj))
+                        if (!this.selections.includes(obj)) {
```

Comment:
Did you mean to also set new_objs = true in this if-statement?

Reply:
Yes, good catch